src/content/en/tools/lighthouse/index.md

```diff
@@ -70,103 +76,138 @@ failing tests as indicators on what you can do to improve your app.
-     data-category="crx-install" data-label="lighthouse-install-button"
-     href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk"
-     title="Install Lighthouse Chrome Extension" target="_blank">
-    <span class="material-icons" style="vertical-align:middle">file_download</span>
-    Lighthouse Chrome Extension</a> from the Chrome Webstore.
+1. Install the <a class="gc-analytics-event" data-category="Lighthouse"
+   data-label="Home / Install Extension" href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk"
+   title="Install Lighthouse Chrome Extension" target="_blank">Lighthouse
+   Chrome Extension</a> from the Chrome Webstore.
+
+[desktop]: https://www.google.com/chrome/browser/desktop/
+
+To run an audit:
+
+1. In Chrome, go to the page you want to audit.
+
+1. Click **Lighthouse** ![Lighthouse][icon]{:.lighthouse-inline}. It should
+   be next to the Chrome address bar. If not, open Chrome's main menu and
+   access it at the top of the menu. After clicking, the Lighthouse
+   menu expands.
+
+     <figure>
+       <img src="images/menu.png" alt="The Lighthouse menu.">
+       <figcaption>
+         <b>Figure 1</b>. The Lighthouse menu
+       </figcaption>
+     </figure>
 
-### Install the Node Command line tool
+[icon]: images/lighthouse-icon-16.png
 
-To install the Node module, do the following:
+1. Click **Generate report**. Lighthouse runs its audits against the
+   currently-focused page, then opens up a new tab with a report of the
+   results.
 
-1. Install [Node](https://nodejs.org), version 5 or greater.
-- Install Lighthouse as a global module:
+     <figure>
+       <img src="images/report.png" alt="The Lighthouse menu.">
+       <figcaption>
+         <b>Figure 2</b>. A Lighthouse report
+       </figcaption>
+     </figure>
+
+### Install and run the Node command line tool {: #cli }
+
+To install the Node module:
+
+1. Download [Google Chrome for Desktop][desktop].
+1. Install the current [Long-Term Support](https://github.com/nodejs/LTS)
+   version of [Node](https://nodejs.org).
+1. Install Lighthouse. The `-g` flag installs it as a global module.
 
         npm install -g lighthouse
 
-Then, run Lighthouse against a URL to audit the page:
+To run an audit:
 
-    lighthouse https://airhorner.com/
+    lighthouse <url>
 
-Pass the `--help` flag to see available options.
+To see audit options:
 
     lighthouse --help
 
-Note: Lighthouse can also be used programmatically as a Node module. See
-[this example][example] for how call Lighthouse to create custom reports, run
-your own audits, and integrate with Mocha testing.
+#### Run the Node module programmatically {: #programmatic }
+
+See [lighthouse-tests.js][programmatic] for an example of running Lighthouse
+programmatically, as a Node module.
+
+[programmatic]: https://github.com/justinribeiro/lighthouse-mocha-example/blob/master/test/lighthouse-tests.js
```

Comment:
you can link to our readme now!

https://github.com/GoogleChrome/lighthouse#using-programmatically